### PR TITLE
docs: update sensor calibration documentation

### PR DESCRIPTION
**Pull request by ismetatabay**
## Description

This PR adds document section for sensor calibration. 

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The Reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Comment by ismetatabay** [comment]
Hi @mitsudome-r san, also this PR is ready for review, FYI.

File: docs/how-to-guides/integrating-autoware/creating-vehicle-and-sensor-description/calibrating-sensors/lidar-lidar-calibration.md [hunk 1]
```diff
@@ -0,0 +1,475 @@
...
+    Storage id:        sqlite3
+    Duration:          112.702s
+    Start:             Sep  5 2023 11:23:51.105 (1693902231.105)
+    End:               Sep  5 2023 11:25:43.808 (1693902343.808)
+    Messages:          2256
+    Topic information: Topic: /sensing/lidar/front/pointcloud_raw | Type: sensor_msgs/msg/PointCloud2 | Count: 1128 | Serialization Format: cdr
+                       Topic: /sensing/lidar/top/pointcloud_raw | Type: sensor_msgs/msg/PointCloud2 | Count: 1128 | Serialization Format: cdr
+    ```
+
+## Mapping-based lidar-lidar calibration
```
[the remark this answers is not in the document]
Comment (mitsudome-r) [reply docs/how-to-guides/integrating-autoware/creating-vehicle-and-sensor-description/calibrating-sensors/lidar-lidar-calibration.md:31]: Same goes with other calibrations pages(lidar-camera, interinsic-camera-calibration, ground-lidar-calibration)

File: docs/how-to-guides/integrating-autoware/creating-vehicle-and-sensor-description/calibrating-sensors/calibrating-sensors.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
+# Starting with TIER IV's CalibrationTools
+
+## Overview
+
+Autoware expects to have multiple sensors attached to the vehicle as input to perception,
+localization,
+and planning stack.
+These sensors must be calibrated correctly,
+and their positions must be defined at `sensor_kit_description` and `individual_params` packages.
+In order to do that, we will use TIER IV' [CalibrationTools](https://github.com/tier4/CalibrationTools) repository.
```
[the remark this answers is not in the document]
Comment (ismetatabay) [reply docs/how-to-guides/integrating-autoware/creating-vehicle-and-sensor-description/calibrating-sensors/calibrating-sensors.md]: Done :+1:

File: docs/how-to-guides/integrating-autoware/creating-vehicle-and-sensor-description/calibrating-sensors/manual-calibration.md [hunk 3]
```diff
@@ -0,0 +1,338 @@
+# Manual calibration for all sensors
+
+## Overview
+
+In this section, we will use [Extrinsic Manual Calibration](https://github.com/tier4/CalibrationTools/blob/tier4/universe/sensor/docs/how_to_extrinsic_manual.md)
+for extrinsic calibrating our sensors.
```
[the remark this answers is not in the document]
Comment (ismetatabay) [reply docs/how-to-guides/integrating-autoware/creating-vehicle-and-sensor-description/calibrating-sensors/manual-calibration.md]: Done :+1:

File: docs/how-to-guides/integrating-autoware/creating-vehicle-and-sensor-description/calibrating-sensors/manual-calibration.md [hunk 4]
```diff
@@ -0,0 +1,338 @@
...
+At this point, we can't completely accurate calibrations results for usage,
```
[the remark this answers is not in the document]
Comment (ismetatabay) [reply docs/how-to-guides/integrating-autoware/creating-vehicle-and-sensor-description/calibrating-sensors/manual-calibration.md]: Done :+1:

File: docs/how-to-guides/integrating-autoware/creating-vehicle-and-sensor-description/calibrating-sensors/manual-calibration.md [hunk 5]
```diff
@@ -0,0 +1,338 @@
...
+but we will have an initial calibration for other tools.
+For example, in the lidar-lidar or camera-lidar calibration phase,
+we will need initial calibration for getting accurate and successful calibration results.
+
+We need a sample bag file for the calibration process
+which includes raw lidar topics and camera topics.
+So our tutorial_vehicle's recorded topics should like this:
```
[the remark this answers is not in the document]
Comment (ismetatabay) [reply docs/how-to-guides/integrating-autoware/creating-vehicle-and-sensor-description/calibrating-sensors/manual-calibration.md]: Done :+1:

File: docs/how-to-guides/integrating-autoware/creating-vehicle-and-sensor-description/calibrating-sensors/lidar-lidar-calibration.md [hunk 6]
```diff
@@ -0,0 +1,475 @@
+# Lidar-Lidar calibration
+
+## Overview
+
+In this tutorial,
+we will explain lidar-lidar calibration over [mapping-based lidar-lidar calibration tool](https://github.com/tier4/CalibrationTools/blob/tier4/universe/sensor/docs/how_to_extrinsic_mapping_based.md) of TIER IV's
+CalibrationTools.
+Also,
+[map-based lidar-lidar calibration method](https://github.com/tier4/CalibrationTools/blob/tier4/universe/sensor/docs/how_to_extrinsic_map_based.md) included in the TIER IV's tools.
```
[the remark this answers is not in the document]
Comment (ismetatabay) [reply docs/how-to-guides/integrating-autoware/creating-vehicle-and-sensor-description/calibrating-sensors/lidar-lidar-calibration.md]: Done :+1:

File: docs/how-to-guides/integrating-autoware/creating-vehicle-and-sensor-description/calibrating-sensors/manual-calibration.md [hunk 7]
```diff
@@ -0,0 +1,338 @@
...
+cd <YOUR-OWN-AUTOWARE-DIRECTORY>/src/autoware/calibration_tools/sensor
+cd extrinsic_calibration_manager/launch
+mkdir <YOUR-OWN-SENSOR-KIT-NAME> # i.e. for our guide, it will ve mkdir tutorial_vehicle_sensor_kit
+cd <YOUR-OWN-SENSOR-KIT-NAME> # i.e. for our guide, it will ve cd tutorial_vehicle_sensor_kit
+touch manual.launch.xml manual_sensor_kit.launch.xml
+```
+
+The created `manual.launch.xml` and `manual_sensor_kit.launch.xml` are version of sample sensor kit
+[aip_x1](https://github.com/tier4/CalibrationTools/tree/tier4/universe/sensor/extrinsic_calibration_manager/launch/aip_x1)
+provided from TIER IV.
```
[the remark this answers is not in the document]
Comment (ismetatabay) [reply docs/how-to-guides/integrating-autoware/creating-vehicle-and-sensor-description/calibrating-sensors/manual-calibration.md]: Done :+1:

File: docs/how-to-guides/integrating-autoware/creating-vehicle-and-sensor-description/calibrating-sensors/intrinsic-camera-calibration.md [hunk 8]
```diff
@@ -0,0 +1,118 @@
+# Intrinsic camera calibration
+
+## Overview
+
+Intrinsic camera calibration is the process
+of determining the internal parameters of a camera that affect how it captures images.
```
[the remark this answers is not in the document]
Comment (ismetatabay) [reply docs/how-to-guides/integrating-autoware/creating-vehicle-and-sensor-description/calibrating-sensors/intrinsic-camera-calibration.md]: Done :+1:

File: docs/how-to-guides/integrating-autoware/creating-vehicle-and-sensor-description/calibrating-sensors.md
[the remark this answers is not in the document]
Comment (ismetatabay) [reply docs/how-to-guides/integrating-autoware/creating-vehicle-and-sensor-description/calibrating-sensors.md:1]: :+1:, I kept in: https://autowarefoundation.github.io/autoware-documentation/pr-448/how-to-guides/integrating-autoware/creating-vehicle-and-sensor-description/calibrating-sensors/

File: docs/how-to-guides/integrating-autoware/creating-vehicle-and-sensor-description/calibrating-sensors/calibrating-sensors.md [hunk 9]
```diff
@@ -0,0 +1,118 @@
...
+```
+
+Then build the all packages
+after the all necessary changes are made on sensor model and vehicle model.
+
+```bash
+colcon build --symlink-install --cmake-args -DCMAKE_BUILD_TYPE=Release
+```
+
+## Usage of CalibrationTools
```
[the remark this answers is not in the document]
Comment (ismetatabay) [reply docs/how-to-guides/integrating-autoware/creating-vehicle-and-sensor-description/calibrating-sensors/calibrating-sensors.md]: I moved other calibration tools on overview page as well: https://autowarefoundation.github.io/autoware-documentation/pr-448/how-to-guides/integrating-autoware/creating-vehicle-and-sensor-description/calibrating-sensors/

File: docs/how-to-guides/integrating-autoware/creating-vehicle-and-sensor-description/calibrating-sensors/lidar-lidar-calibration.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (ismetatabay) [reply docs/how-to-guides/integrating-autoware/creating-vehicle-and-sensor-description/calibrating-sensors/lidar-lidar-calibration.md]: I applied new structure according to the your suggestion:

Each calibration method has similar structure:

image

File: docs/how-to-guides/integrating-autoware/creating-vehicle-and-sensor-description/calibrating-sensors/lidar-camera-calibration.md [hunk 10]
```diff
@@ -0,0 +1,447 @@
...
+in order to create a comprehensive and coherent representation of the environment
+by projecting lidar point onto camera image.
+At this tutorial,
+we will explain [TIER IV's interactive camera calibrator](https://github.com/leo-drive/CalibrationTools/blob/golf_test/sensor/docs/how_to_extrinsic_interactive.md).
+Also, If you have aruco marker boards for calibration,
+another [Lidar-Camera calibration method](https://github.com/leo-drive/CalibrationTools/blob/golf_test/sensor/docs/how_to_extrinsic_tag_based.md) is included in TIER IV's CalibrationTools repository.
+
+!!! warning
+
+    Please get initial calibration results from [Manual Calibration](./manual-calibration.md) section, it is important for getting accurate results from this tool.
```
[the remark this answers is not in the document]
Comment (ismetatabay) [reply docs/how-to-guides/integrating-autoware/creating-vehicle-and-sensor-description/calibrating-sensors/lidar-camera-calibration.md]: I added :+1: .

**Comment by ismetatabay** [comment]
Hi @mitsudome-r san, thank you for your suggestions. I applied your review comments on documentation, can you check it when you are available?

**Review by mitsudome-r — APPROVED** [review]
LGTM

**Comment by mitsudome-r** [comment]
@ismetatabay Thank you for updating the documentation. The instruction generally looks good to me.
I was wondering if you were also planning a sample rosbag to go through the instructions or not. If you have sample rosbags, then I can upload them to AWF's cloud and add the link to the instructions.

**Comment by ismetatabay** [comment]
Hi @mitsudome-r, that sounds great. We can provide a sample rosbag for the calibration process. I have included to [drive link](https://drive.google.com/drive/folders/1XyVOUdyjUrwhUw7gIZ1g2zU6r8LhLUMx?usp=drive_link) two bag files for ground-lidar calibration and lidar-lidar calibration. Could you check the [link](https://drive.google.com/drive/folders/1XyVOUdyjUrwhUw7gIZ1g2zU6r8LhLUMx?usp=drive_link)?

We performed the calibration process with these bags and this [calibration tools branch](https://github.com/leo-drive/tutorial_vehicle_calibration_tools/tree/tutorial_vehicle) for tutorial_vehicle.

**Pull request merged**